logstash-core/lib/logstash/config/loader.rb

```diff
@@ -0,0 +1,99 @@
+require "logstash/config/defaults"
+
+module LogStash; module Config; class Loader
+  def initialize(logger, is_config_test)
+    @logger = logger
+    @is_config_test = is_config_test
```

Comment:
+1 on changing this, I see no reason not to show this when debug is enabled, even if it's not a configuration test.